Backend/ServiceLayer/Services/PointService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using DataAccessLayer.Database;
+using DataAccessLayer.Models;
+using DataAccessLayer.Repositories;
+
+namespace ServiceLayer.Services
+{
+    public class PointService : IPointService
+    {
+        private PointRepository _PointRepo;
+
+        public PointService()
+        {
+            _PointRepo = new PointRepository();
+        }
+
+        public Point CreatePoint(DatabaseContext _db, Point point)
+        {
+            if (_PointRepo.ExistingPoint(_db, point))

```

Comment:
This doesn't exactly make sense. This only checks by ID - it's really not validating anything. See my comment on this method in the point repository

Reply:
I removed this if statement.